modules/cluster/src/main/java/io/fluo/cluster/mini/MiniAdmin.java

```diff
@@ -42,6 +43,8 @@ public static void main(String[] args) {
         System.exit(-1);
       }
       options.validateConfig();
+      
+      ClusterUtil.verifyConfigFilesExist(options.getConfigDir(), "fluo.properties", "log4j-stdout.xml");
```

Comment:
For the log config files, could verify their existance in `Log4jUtil.init()` and `LogbackUtil.ini()`.  Then the code calling those methods would not have to make assumptions.